conditional/blueprints/housing.py

```diff
@@ -11,6 +11,7 @@
 from conditional.util.ldap import ldap_get_member
 from conditional.util.ldap import ldap_get_roomnumber
 from conditional.util.ldap import ldap_get_current_students
+from conditional.util.ldap import _ldap_add_member_to_group as ldap_add_member_to_group
```

Comment:
If we're going to make this method public, let's remove the `_` instead of aliasing it.